storage-proofs/post/src/fallback/compound.rs

```diff
@@ -234,6 +238,7 @@ mod tests {
                 sector_size: sector_size as u64,
                 challenge_count,
                 sector_count,
+                is_winning: false,
```

Comment:
For the sake of getting this working quickly, I ended up specializing all the tests to Window PoSt (`is_winning: false`). To be thorough, they should probably be extended to exercise both code paths.